src/Ucommerce.Sitecore/SitecoreConstants.cs

```diff
@@ -39,10 +39,10 @@ internal static class SitecoreConstants
 
 		public const string SitecoreDataProviderTreeServiceId = "TreeServiceContentEditor";
 
-	    public const string RewriteCategoryProduct = "categoryProduct";
-        public const string RewriteProduct = "product";
-        public const string RewriteCategory = "category";
         public const string RewriteCatalog = "catalog";
+        public const string RewriteCategory = "category";
+	    public const string RewriteCategoryProduct = "product";
```

Comment:
Let's bring back simpler name 🙏 
```suggestion
	    public const string RewriteProduct = "product";
```